swri_yaml_util/CMakeLists.txt

```diff
@@ -28,10 +28,10 @@ configure_file(version.h.in ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_INCLUDE_DEST
 find_package(Boost REQUIRED)
 
 include_directories(include
-  ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_INCLUDE_DESTINATION}
-  ${catkin_INCLUDE_DIRS}
   ${Boost_INCLUDE_DIRS}
+  ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_INCLUDE_DESTINATION}
   ${YamlCpp_INCLUDE_DIR}
+  ${catkin_INCLUDE_DIRS}
```

Comment:
Good point.  Both `swri_yaml_util` and `swri_transform_util` didn't have `${catkin_INCLUDE_DIRS}` second, so I fixed them.